src/me/goddragon/teaseai/api/session/Session.java

```diff
@@ -24,6 +41,21 @@
     private boolean haltSession = false;
     private long startedAt;
 
+    private EstimAPI estimAPI = null;
+    private List<Mode> estimEnabledModes = null;
+    private Map<Channel, Integer> estimChannelValues = new HashMap<Channel, Integer>();
+    
+    private List<Mode> string2List(String string) {
```

Comment:
Own class for this? Util class? Estim Class?

Reply:
Done. It is now in EstimState.